packages/strapi-plugin-content-manager/admin/src/containers/RBACManager/useSyncRbac.js

```diff
@@ -0,0 +1,33 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import pluginId from '../../pluginId';
+import { resetPermissions, setPermissions } from './actions';
+
+const selectPermissions = state => state.get(`${pluginId}_rbacManager`).permissions;
+
+const selectCollectionTypePermissions = state =>
+  state.get('permissionsManager').collectionTypesRelatedPermissions;
+
+const useSyncRbac = (query, collectionTypeUID = 'listView') => {
```

Comment:
ℹ️ This is basically the RBACManager in a hook fashion

Reply:
Why do you set the collectionTypeUID to listview I don't think it's needed.